src/bluehawk/actions/ConsoleActionReporter.ts

```diff
@@ -33,63 +33,72 @@ export class ConsoleActionReporter implements ActionReporter {
     return this._count.errors;
   }
 
-  onBinaryFile(event: FileEvent): void {
+  onBinaryFile = (event: FileEvent): void => {
```

Comment:
Forgot to use arrow notation. Class method style messes with spread operator on class instance.